### PR TITLE
Allow running manila kuttl test in any namespace

### PR DESCRIPTION
**Pull request by vhariria**

### CODE REVIEW

**Comment by vhariria** [comment]
recheck

File: Makefile [hunk 1]
```diff
@@ -308,15 +308,16 @@ DATAPLANE_KUTTL_NAMESPACE ?= dataplane-kuttl-tests
...
+MANILA_BRANCH           ?= main
+MANILA                  ?= config/samples/manila_v1beta1_manila.yaml
+MANILA_CR               ?= ${OPERATOR_BASE_DIR}/manila-operator/${MANILA}
 # TODO: Image customizations for all Manila services
-MANILA_KUTTL_CONF     ?= ${OPERATOR_BASE_DIR}/manila-operator/kuttl-test.yaml
-MANILA_KUTTL_DIR      ?= ${OPERATOR_BASE_DIR}/manila-operator/tests/kuttl/tests
-MANILA_KUTTL_TIMEOUT  ?= 180
+MANILA_KUTTL_CONF       ?= ${OPERATOR_BASE_DIR}/manila-operator/kuttl-test.yaml
+MANILA_KUTTL_DIR        ?= ${OPERATOR_BASE_DIR}/manila-operator/tests/kuttl/tests
+MANILA_KUTTL_TIMEOUT    ?= 180
```
Comment (abays) [inline Makefile:319]: Is this variable used anywhere?

Comment (vhariria) [reply Makefile:319]: Occasionally we use it for triage, currently not used in the file. Can be removed until needed.

Comment (abays) [reply Makefile:319]: Perhaps we can remove it for now, otherwise it might give the user the impression that it actually currently does something.

Comment (vhariria) [reply Makefile]: No problem, done.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**